standalone/include/psen_scan_v2_standalone/data_conversion_layer/laserscan_conversions.h

```diff
@@ -115,7 +115,7 @@ LaserScanConverter::calculateMaxAngle(const std::vector<data_conversion_layer::m
       std::accumulate(frames.begin(), frames.end(), uint16_t{ 0 }, [](uint16_t total, const auto& frame) {
         return total + frame.measurements().size();
       });
-  return min_angle + resolution * static_cast<int>(number_of_samples);
+  return min_angle + resolution * static_cast<int>(number_of_samples - 1);
```

Comment:
it seem like you are changing it at two places now. You change it at the sending of the request and here. why?

Reply:
This position in the code only affects the calculation of the laser scan message and has nothing to do with the start request